recipes/efa_install.rb

```diff
@@ -48,6 +48,11 @@
   # efa-kmod currently unavailable for ARM instances
   installer_options += " -k"
 end
+if efa_gdr_enabled?
```

Comment:
by executing rubocop in the recipes the suggestion is to use a single line if.

```ruby
# enable gpudirect support
installer_options += " -g" if efa_gdr_enabled?
```

Let's do it for the previous line too.